example/fixtures.py

```diff
@@ -130,20 +155,100 @@ def containers(request):
     'beat_container', ['single_container', 'beat_only_container'])
 
 
+def volume_fixture(volume, name, scope='function'):
+    @pytest.fixture(name=name, scope=scope)
+    def fixture(docker_helper):
+        # FIXME: DockerHelper volumes API
+        volume = docker_helper._client.volumes.create(name, driver='local')
+        yield volume
+        volume.remove()
+    return fixture
+
+
+static_volume = volume_fixture('static', 'static_volume')
+media_volume = volume_fixture('media', 'media_volume')
+gunicorn_volume = volume_fixture('gunicorn', 'gunicorn_volume')
+
+
+class NginxContainer(ContainerBase):
+    def __init__(
+            self, name, image, static_volume, media_volume, gunicorn_volume):
+        super().__init__(name, image)
+        self._static_volume = static_volume
+        self._media_volume = media_volume
+        self._gunicorn_volume = gunicorn_volume
+
+    def create_kwargs(self):
+        return {
+            'ports': {'80/tcp': ('127.0.0.1',)},
+            'volumes': {
+                self._static_volume.id: {
+                    'bind': '/usr/share/nginx/static',
+                    'mode': 'ro',
+                },
+                self._media_volume.id: {
+                    'bind': '/usr/share/nginx/media',
+                    'mode': 'ro',
+                },
+                self._gunicorn_volume.id: {
+                    'bind': '/var/run/gunicorn',
+                    'mode': 'rw',
+                },
+            }
+        }
+
+    def wait_for_start(self, docker_helper, container):
+        # No real way to know when Nginx is ready. Wait a short moment.
+        time.sleep(0.5)
```

Comment:
A better option may be to make some dummy requests against Nginx.

Reply:
Does it have a status page we can hit?